previews/aws-lambda-simple-queue-service.md

```diff
@@ -41,6 +41,16 @@ snippet: configure-routing
 
 snippet: custom-diagnostics
 
+### Delayed Retries
+
+[Delayed retries](/nservicebus/recoverability/configure-delayed-retries.md) are disabled by default when using AWS Lambdas. Delayed retries can be configured using the following snippet:
+
+snippet: delayed-retries
+
+If the time increase is expected to be greater than [15 minutes](/transports/sqs/delayed-delivery.md#enable-unrestricted-delayed-delivery), it might be required to enable `UnrestrictedDurationDelayedDelivery` on the endpoint:
```

Comment:
This will probably require creating the Fifo queue as well.

Reply:
Added